### PR TITLE
Ear 684 disable skip logic on first question

### PR DESCRIPTION
**Pull request by jchapman68**
### What is the context of this PR?

> https://collaborate2.ons.gov.uk/jira/browse/EAR-684
> 
> 

### How to review 

> Add to the list below as appropriate, including screenshots when necessary

* Create a questionnaire with a number of questions and at least two sections
* Check the Add skip logic button is disabled only for first question in the questionnaire
* Create a skip logic on second question
* Move question to become first below and check skip condition has been deleted
* Create skip logic on second question
* Delete first question
* Check the skip has been removed on the new first question
* Create skip logic on first question in second section
* Move second section to become first section
* Check skip logic has been removed from first question of first section
* Create skip logic on first question of second section
* Delete the first section
* Check the skip logic has been deleted from first question in first section

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Jira ticket for this task into the next stage of the process

### CODE REVIEW

File: eq-author/src/App/page/Logic/SkipLogic/SkipLogicPage/NoSkipConditions.js [hunk 1]
```diff
@@ -34,34 +34,40 @@ const Paragraph = styled.p`
...
-    </AddSkipConditionsButton>
+    <Title>{isFirstQuestion ? "Skip logic not available for this question" : "No skip conditions exist for this question" }</Title>
+    <Paragraph>{isFirstQuestion ? "You can't add skip logic to the first question in a questionnaire." : "All users will see this question if no skip logic is added."}</Paragraph>
+      <AddSkipConditionsButton
+        small
+        naked
+        variant="primary"
+        onClick={onAddSkipCondtions}
+        data-test="btn-add-skip-condition"
+        isFirstQuestion={isFirstQuestion}
```
Comment (tjamesmac) [inline eq-author/src/App/page/Logic/SkipLogic/SkipLogicPage/NoSkipConditions.js:57]:
```suggestion
        isFirstQuestion={isFirstQuestion}
        disabled={isFirstQuestion}
```

**Review by tjamesmac — APPROVED** [review]
Nice one

**Review by tbaggaley — COMMENTED** [review]
Looks good Josh, just one question. The ticket says that when you remove a question in front and another question with skip logic becomes the first in the questionnaire, "then the skip logic is removed and the button is disabled".

The skip logic isn't being removed when questions become first. But when you remove it the changes work as intended (the new message is displayed etc.)

Just checking if this is needed - should we also remove the skip logic from these questions? Otherwise happy to approve

**Comment by tjamesmac** [comment]
@tbaggaley yes it is needed good spot!

**Review by tbaggaley — APPROVED** [review]
Looking good; skip conditions are being deleted now on page/section move/delete

**Review by tjamesmac — DISMISSED** [review]

**Review by MattElcock — CHANGES_REQUESTED** [review]
Ignore this. I didn't have the latest changes.

**Review by MattElcock — DISMISSED** [review]
Sorry about my earlier review. I didn't have the latest changes. It all looks good. Well done. :)

**Review by jasehumphr — APPROVED** [review]
All good

**Review by tbaggaley — APPROVED** [review]

**Pull request merged**